### PR TITLE
[docs] update tsdoc comment.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- update tsdoc comment.

### CODE REVIEW

File: src/nft.ts [hunk 1]
```diff
@@ -169,14 +169,14 @@ export default class Nft extends FactoryBase {
   }
 
   /**
-   * Searches for AINFT objects created on the AIN Blockchain.
+   * Searches for AINFT objects created on the AIN Blockchain. This method accesses public data only.
```
Comment (platfowner) [inline src/nft.ts:172]: how about mentioning about the signature verification? e.g.
"This method accesses public data only and does not require signature in the requests."

File: src/nft.ts [hunk 2]
```diff
@@ -206,13 +206,13 @@ export default class Nft extends FactoryBase {
   }
 
   /**
-   * Searches for AINFTs on the ain blockchain.
+   * Searches for AINFTs on the ain blockchain. This method accesses public data only.
```
Comment (platfowner) [inline src/nft.ts:209]: ditto?

File: src/nft.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/nft.ts]: done!

File: src/nft.ts (hunk 2 above)
Comment (jiyoung-an) [reply src/nft.ts]: done!

**Comment by jiyoung-an** [comment]
Thank you for helpful comment! @platfowner now merging...

**Pull request merged**